docs/how-to-contribute.md

```diff
@@ -53,7 +53,7 @@ To add a new blog post to the gridsome.org blog:
 - Add a new folder following the pattern `/blog/yyyy-mm-dd-title` (for example, 2018-09-14-say-hello-to-gridsome). Within this newly created folder add an `index.md` file.
 - Add `title`, `date`, `author`, and `tags` to the frontmatter of your `index.md`.
 - If your blog post contains images add them to your blog post folder and reference them in your post's `index.md`.
-- Ensure any links to **gridsome.org pages** are relative links - `/docs/how-to-contribute` instead of `https://gridsome.org/docs/how-to-contribute`
+- Ensure any links to **gridsome.org pages** are relative links - `/docs/how-to-contribute/` instead of `https://gridsome.org/docs/how-to-contribute/`
 - Commit and push to your fork
 - Create a pull request from your branch
   - We recommend using a prefix of `docs`, like `docs/your-change`.
```

Comment:
Also ending slash here?

Reply:
`docs/your-change` refers to a git branch name, so we don't need the trailing slash here.